rasa/core/featurizers/single_state_featurizer.py

```diff
@@ -292,11 +292,13 @@ def encode_entities(
             # we cannot build a classifier if there are less than 2 class
             return {}
 
-        parsed_text = interpreter.featurize_message(Message({TEXT: entity_data[TEXT]}))
+        parsed_text = (
+            interpreter.featurize_message(Message({TEXT: entity_data[TEXT]})) or {}
+        )
```

Comment:
I'd rather do afterwards
```
if parsed_text is None:
    return {}
```

Reply:
why?

Reply:
I think returning `{}` instead of `{ENTITY_TAGS: Features(...)}` with zero-features saves some bandwidth and computation time.

Reply:
I've made that change as vova suggested.